cmd/scan.go

```diff
@@ -89,6 +90,7 @@ func scan(cmd *cobra.Command, args []string) {
 	disableFlowSeperationByDataElement, _ := cmd.Flags().GetBool("disable-flow-separation-by-data-element")
 	disable2ndLevelClosure, _ := cmd.Flags().GetBool("disable-2nd-level-closure")
 	generateUnresolvedNameReport, _ := cmd.Flags().GetBool("generate-unresolved-name-report")
+	testOutput, _ := cmd.Flags().GetBool("test-output")
```

Comment:
Rename the variable in accordance with the new flag name.

Reply:
Done @ojaswa1942